docs/django.rst

```diff
@@ -25,6 +25,9 @@ I strongly recommend not using
 unless you really have to.
 Because Hypothesis runs this in a loop the performance problems it normally has
 are significantly exacerbated and your tests will be really slow.
+If you are using :class:`~hypothesis.extra.django.TransactionTestCase`,
```

Comment:
It's probably worth noting that this might be a useful thing to do if you're using `hypothesis.extra.django.TestCase` too - it's not as slow as TransactionTestCase, but it's still pretty slow.

Reply:
I can see your point, but I'd rather not - if someone actually has the problem they'll probably read this anyway, and if not I prefer to leave checks enabled.

Reply:
Fine by me.